website/src/categories/components/toggle-group/usage.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Toggle button usage
+tags: usage
+eleventyNavigation:
+  parent: Toggle button
+  key: ToggleButtonUsage
+---
+
+<section>
+
+<div class="ds-example" style="flex-direction: column; gap: 2rem;">
+<sl-toggle-group multiple>
+<sl-toggle-button aria-label="Bold" pressed>
+<sl-icon name="far-bold" slot="default"></sl-icon>
+<sl-icon name="fas-bold" slot="pressed"></sl-icon>
+</sl-toggle-button>
+<sl-toggle-button aria-label="Italic">
+<sl-icon name="far-italic" slot="default"></sl-icon>
+<sl-icon name="fas-italic" slot="pressed"></sl-icon>
+</sl-toggle-button>
+<sl-toggle-button aria-label="Underline" pressed>
+<sl-icon name="far-underline" slot="default"></sl-icon>
+<sl-icon name="fas-underline" slot="pressed"></sl-icon>
+</sl-toggle-button>
+</sl-toggle-group>
+<sl-toggle-group>
+  <sl-toggle-button>All types</sl-toggle-button>
+  <sl-toggle-button>Primary education</sl-toggle-button>
+  <sl-toggle-button pressed>Secondary education</sl-toggle-button>
+</sl-toggle-group>
+</div>
+
+<div class="ds-code">
+
+  ```html
+    <sl-toggle-group multiple>
+      <sl-toggle-button aria-label="Bold" pressed>
+        <sl-icon name="far-bold" slot="default"></sl-icon>
+        <sl-icon name="fas-bold" slot="pressed"></sl-icon>
+      </sl-toggle-button>
+      <sl-toggle-button aria-label="Italic">
+        <sl-icon name="far-italic" slot="default"></sl-icon>
+        <sl-icon name="fas-italic" slot="pressed"></sl-icon>
+      </sl-toggle-button>
+      <sl-toggle-button aria-label="Underline" pressed>
+        <sl-icon name="far-underline" slot="default"></sl-icon>
+        <sl-icon name="fas-underline" slot="pressed"></sl-icon>
+      </sl-toggle-button>
+    </sl-toggle-group>
+    <sl-toggle-group>
+      <sl-toggle-button>All types</sl-toggle-button>
+      <sl-toggle-button>Primary education</sl-toggle-button>
+      <sl-toggle-button pressed>Secondary education</sl-toggle-button>
+    </sl-toggle-group>
+  ```
+
+</div>
+
+</section>
+
+<section>
+
+## When to use
+Toggle group are best used in situations where users need to make a single selection from multiple options within a defined group. Here are two common scenarios:
+
+### Related Exclusive Choices
+Use a toggle group when offering a set of options where only one can be active at a time. This is ideal for scenarios like mode selection, where choosing one option automatically deselects others (e.g., light vs. dark mode). The toggle group ensures clear feedback, allowing users to easily understand which option is currently active.
```

Comment:
We should make clearer that this is just a visual change; if you have the same data but it looks differently you can use a toggle group, if you load different data altogether you need to use tabs or navigation links.